cmd/kubeadm/app/cmd/init.go

```diff
@@ -396,6 +399,14 @@ func newInitData(cmd *cobra.Command, args []string, options *initOptions, out io
 	}, nil
 }
 
+func union(lists ...[]string) []string {
```

Comment:
i think we had a similar utility function in another location.

Reply:
I'd be interested in it if you find it again. `git grep`-ing for `func Union`/`func union` didn't yield anything.

Reply:
Alternatively, I could move this to some utility file. Any preference on the location? (I was thinking of moving it under `cmd/kubeadm/app/util/sets.go`)

Reply:
you can do in the code above something like this:
```go

ignorePreflightErrors := sets.NewString(options.ignorePreflightErrors).Union(sets.NewString(cfg.NodeRegistration.IgnorePreflightErrors)).List()
```

Reply:
Yes, I thought of this too, but found it relatively verbose, so thought I'd hide the details behind a function. I can do this if there is a strong opinion though. Thanks @kad for pointing this out in any case 👍

Reply:
Resolved now that the union is done directly by `ValidateIgnorePreflightErrors` as suggested in https://github.com/kubernetes/kubernetes/pull/75499#issuecomment-492216224.